City-of-Helsinki/employment-services-ui
# Thf 491 breadcrumb fix

Fixed article (news) pages breadcrumbs. 

How to test:

-Go some news page example http://localhost:3000/ajankohtaista/uutiset/helsinki-jarjesti-ensimmaisen-rekrytointitapahtuman-matkailutoimialoille
- open same news page on production  https://tyollisyyspalvelut.hel.fi/ajankohtaista/uutiset/helsinki-jarjesti-ensimmaisen-rekrytointitapahtuman-matkailutoimialoille
- See that the breadcrumbs should be :
- Front page > Current Matters > News > “Name of the Article”
not below
-Front page > “Name of the Article”
